src/pages/news/2020-03-09-tbtc-101-1.it.md

```diff
@@ -0,0 +1,43 @@
+---
+template: news-item
+title: tBTC 101
+description: tBTC 101
+date: 2020-03-09T20:35:00.000Z
+---
+tBTC è una sidechain progettata per permettere ai BTC holders di guadagnare in modo trustless usando le applicazioni di finanza decentralizzata (DeFi)
+
+##### 3 steps per guadagnare con i tuoi BTC
+
+[TBTC](http://tbtc.network) è un ERC-20 token totalmente garantito da BTC. Attualmente [live sulla testnet Ropsten](https://tbtc.network/news/2020-02-14-ropsten), TBTC permette agli utenti di usare BTC sulla DeFi per finanziare prestiti, guadagnare interessi in BTC e molto altro ancora.
+
+![](https://lh3.googleusercontent.com/Hcbbww1HmPmVgYNdOPMaZzaxCLfrs0tBg_HOYv-LVPRrf-wW-F4CEPTQHNC8yJZJE1HZOspoaVAZvYND9E7RYRca7FFNeHWhu1s22SHZnMjKTBhszGSdsuh6_o-R4bsC8fHJJ-Lx)L'obiettivo di tBTC è di dare a BTC la forma di uno smart contract. tBTC rende possibile usare Bitcoin sulla rete Ethereum in 3 semplici passaggi:
```

Comment:
We shouldn't use hot-linked images that we don't control like this- you can use the image from the CMS image library